docs/source/explanation/algorithms/classification/multi_class_classification.rst

```diff
@@ -14,10 +14,42 @@ For the supervised training we use the following algorithms components:
 
-- Additionally, we use `No Bias Decay (NBD) <https://arxiv.org/abs/1812.01187>`_ technique and **early stopping** to add adaptability to the training pipeline and prevent overfitting. Besides this we use `Balanced Sampler <https://github.dev/openvinotoolkit/training_extensions/blob/develop/otx/mpa/modules/datasets/samplers/balanced_sampler.py#L11>`_ to create an efficient batch that consists of balanced samples over classes, reducing the iteration size as well.
-
-
-To further enhance the performance of the algorithm in case when we have a small number of data we use `Supervised Contrastive Learning <https://arxiv.org/abs/2004.11362>`_. More specifically, we train a model with two heads: classification head with Influence-Balanced Loss and SupCon head with `Barlow Twins loss <https://arxiv.org/abs/2103.03230>`_.
+- ``Training technique``
+    - `No Bias Decay (NBD) <https://arxiv.org/abs/1812.01187>`_: To add adaptability to the training pipeline and prevent overfitting.
+    - ``Early stopping``: To add adaptability to the training pipeline and prevent overfitting. You can use early stopping like the below command.
+      
+      .. code-block::
+
+        $ otx train {TEMPLATE} ... \
+            params \
+            --learning_parameters.enable_early_stopping=True \      # is early stopping used
+            --learning_parameters.early_stop_start=3 \              # the number of epochs (iters) in which early stopping proceeds
+            --learning_parameters.early_stop_patience=8 \           # (for epoch runner) stop if the model don't improve within the number of epochs of patience
+            --learning_parameters.early_stop_iteration_patience=8 \ # (for iter runner) stop if the model don't improve within the number of iterations of patience
+
+    - `Balanced Sampler <https://github.dev/openvinotoolkit/training_extensions/blob/develop/otx/mpa/modules/datasets/samplers/balanced_sampler.py#L11>`_: To create an efficient batch that consists of balanced samples over classes, reducing the iteration size as well.
+    - `Supervised Contrastive Learning (SupCon) <https://arxiv.org/abs/2004.11362>`_: To enhance the performance of the algorithm in case when we have a small number of data. More specifically, we train a model with two heads: classification head with Influence-Balanced Loss and contrastive head with `Barlow Twins loss <https://arxiv.org/abs/2103.03230>`_. It enables using `--learning_parameters.enable_supcon=True` in CLI.
+      The below table shows how much performance SupCon improved compared with baseline performance on three baseline datasets with 10 samples per class: CIFAR10, Eurosat-10, and Food-101.
+
+      +-----------------------+---------+---------+------------+---------+----------+---------+
```

Comment:
@sungchul2 I found it while checking the latest built documentation. Is this table location right here? It is marked as Self-SL.

Reply:
I updated it at https://github.com/openvinotoolkit/training_extensions/pull/1756.
Could you check it?

Reply:
I updated table columns at https://github.com/openvinotoolkit/training_extensions/pull/1756.
Could you check it?